### PR TITLE
Apply some general polish to the code/yaml

### PR DESCRIPTION
**Pull request by abcdefg30**
You probably want to review the 3rd commit with `?w=1`: https://github.com/OpenRA/OpenRA/commit/ac70d1ed097c360c499dfc70ffe89d18c7fd40c1?w=1

### CODE REVIEW

**Comment by abcdefg30** [comment]
Updated.

**Comment by abcdefg30** [comment]
Rebased.

**Comment by Mailaender** [comment]
:+1:

File: mods/ts/rules/defaults.yaml [hunk 1]
```diff
@@ -77,9 +77,21 @@
 		UpgradeTypes: notmobile
 		UpgradeMaxEnabledLevel: 0
 
+^CloakableUnit:
```
Comment (penev92) [inline mods/ts/rules/defaults.yaml]: Nit: "CloakableActor", or just "Cloakable" please.

Comment (abcdefg30) [inline mods/ts/rules/defaults.yaml]: Changed it to `Cloakable`.

File: OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs [hunk 2]
```diff
@@ -249,80 +249,83 @@ public void ResolveOrder(Actor self, Order order)
...
+			if (self.World.AllowDevCommands && self.Owner.PlayerActor.Trait<DeveloperMode>().FastBuild)
+				return 0;
+
 			var unit = self.World.Map.Rules.Actors[unitString];
-			if (unit == null || !unit.HasTraitInfo<BuildableInfo>())
+			if (unit == null)
 				return 0;
 
+			var time = unit.GetBuildTime() * Info.BuildSpeed;
+			return (int)time;
```
Comment (obrakmann) [inline OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs]: You could just `return GetBuildTime(unit)` here. Move the null check up and remove the cheat check.

**Comment by obrakmann** [comment]
:+1: to everything except the last commit. You would need to write a `ClassicProductionQueue.GetBuildTime(ActorInfo unit)` as well. Currently this causes the production tooltip in RA to show the wrong build time when you have multiple factories.

**Comment by abcdefg30** [comment]
Dropped the last commit, as it wasn't polish anymore and got complex. I'll file a new pr for that.

**Comment by abcdefg30** [comment]
Here is the other pr:  #10557

**Comment by obrakmann** [comment]
:+1:

**Pull request merged**